supernovacontroller/sequential/i3c.py

```diff
@@ -1378,7 +1378,7 @@ def ccc_unicast_endxfer(self, target_address):
 
         return self._process_response("ccc_unicast_endxfer", responses)
 
-    def ccc_broadcast_setxtime(self, timing_parameter):
+    def ccc_broadcast_setxtime(self, timing_parameter, aditional_data = []):
         """
         Performs a broadcast SETXTIME (Set Extra Timing) operation on the I3C bus.
```

Comment:
According to the MIPI I3C spec, SETXTIME stands for Setting the Exchange Timing, no the Extra Timing.